src/EditorFeatures/CSharpTest/CodeActions/ConvertLinq/ConvertLinqQueryToForEachTests.cs

```diff
@@ -4355,6 +4355,88 @@ IEnumerable<int> enumerable()
             await TestInRegularAndScriptAsync(source, output, parseOptions: new CSharpParseOptions(CodeAnalysis.CSharp.LanguageVersion.CSharp8));
         }
 
+        #endregion
+
+        #region CaretSelection
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertQueryToForEach)]
+        public async Task DeclarationSelection()
```

Comment:
Consider adding a WorkItemAttribute to the added tests so the tests can be linked to the underlying discoverability issue and help any future developers working on the refactoring.